integration/update_test.go

```diff
@@ -466,6 +466,58 @@ func TestUpdateSet(t *testing.T) {
 	}
 }
 
+func TestUpdateTimestamp(t *testing.T) {
+	t.Parallel()
+
+	// store the current timestamp with $currentDate operator;
+	t.Run("currentDateReadBack", func(t *testing.T) {
+		maxDifference := time.Duration(2 * time.Second)
+		now := time.Now()
+		nowTimestamp := primitive.Timestamp{T: uint32(now.Unix()), I: uint32(0)}
```

Comment:
That's the only usage of `now` variable.
```suggestion
		nowTimestamp := primitive.Timestamp{T: uint32(time.Now().Unix()), I: uint32(0)}
```